jvm/src/main/scala/io/glutenproject/execution/HashJoinExecTransformer.scala

```diff
@@ -913,7 +913,7 @@ trait HashJoinLikeExecTransformer
       ConverterUtils.getTypeNode(attr.dataType, attr.nullable)
     }
     Any.pack(TypeBuilder.makeStruct(false,
-      new util.ArrayList[TypeNode](inputTypeNodes.asJava)).toProtobuf)
+      new util.ArrayList[TypeNode](inputTypeNodes.asJava)).toProtobuf, "")
```

Comment:
What is this change for?

Reply:
@marin-ma Do we need the change?

Reply:
The "" is to pass an empty typeUrlPrefix. If we don't provide this arg, it will use "type.googleapis.com" by default, and will fail to parse into json here https://github.com/oap-project/gluten/blob/7732a488daf1c0ff3154b8f43d618b9a02b75f47/cpp/src/compute/exec_backend.h#L61-L72

Reply:
Thanks for the details. Is this a common issue that should be fixed for all the usages like this?

Reply:
I've checked the usage and only Join use the extension node on execution phase. Other operators only use it in validation phase where the plan for validation won't be parsed into json string.